CHANGELOG.md

```diff
@@ -9,6 +9,11 @@ All notable changes to this project will be documented in this file.
 
 #### Fixed
 
+## 4.5.0
+
+#### Fixed
+*  #144 until RxSwift 5.0.0 is released
```

Comment:
how about something like "Updated Carthage minimum Swift version to 4.2"?

Reply:
I think it's just "Minimum Swift version is 4.2 (More details in #144)"

Reply:
Yeah, anything that would say what was updated without me needing to go to the GitHub issue to find out.